Test/SchemaCompilerSpec.hs

```diff
@@ -178,7 +178,7 @@ tests = do
 
                     type instance PrimaryKey "users" = UUID
 
-                    type User = User' 
+                    type User = User'
```

Comment:
My editor is trimming white space, but the white space is needed for the tests. I'll fix it manually for now.

```suggestion
                    type User = User' 
```